repositories/implementations/csv_mlb_player.go

```diff
@@ -0,0 +1,68 @@
+package repositories
+
+import (
+	"encoding/csv"
+	"errors"
+	"log"
+	"os"
+	"strconv"
+
+	e "github.com/EloYaniel/academy-go-q42021/entities"
+)
+
+// CSVMLBPlayerRepository implements MLBPlayerRepository interface
+type CSVMLBPlayerRepository struct{}
+
+func (repo *CSVMLBPlayerRepository) GetMLBPlayers() ([]e.MLBPlayer, error) {
+	f, err := os.Open("data/mlb_players.csv")
```

Comment:
i would recommend having this file path as an attribute on your repository struct, that way it will be flexible enough to receive and work with any path which will be necessary when you need to unit test this, because you don't want to use your real file to do tests, you would like to make your function read a test file.

The way you have it now binds your function with only this specific file. You will probably need a "constructor" function that receives that path as an argument and sets it to the new repository instance